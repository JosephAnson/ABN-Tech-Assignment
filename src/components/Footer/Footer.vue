<template>
  <div
    class="mt-5 mx-auto text-center opacity-75 dark:opacity-50 text-sm py-10"
  >
    Joseph Anson - ABN Tech Assignment
  </div>
</template>
