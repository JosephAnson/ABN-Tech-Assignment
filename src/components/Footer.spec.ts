import { render } from "@testing-library/vue"
import { describe, expect, test } from "vitest"
import Footer from "./Footer.vue"

describe("Footer", () => {
  test("should render footer", () => {
    const { html } = render(Footer)
    expect(html()).toMatchSnapshot()
  })
})
