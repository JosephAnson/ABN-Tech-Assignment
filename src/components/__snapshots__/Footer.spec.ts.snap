// Vitest Snapshot v1

exports[`Footer > should render footer 1`] = `"<div class=\\"mt-5 mx-auto text-center opacity-75 dark:opacity-50 text-sm py-10\\"> Joseph Anson - ABN Tech Assignment </div>"`;
